onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
```

Comment:
I would create a docstring for the module to describe all rewrites involved.

Reply:
Where should I put this information, top of the module, inside the functions? It the documentation is published, that would help. For the time being, my goal to get better performance. The code may change.

Reply:
I'll do it once all patterns are done.